documentation/adoc/cluster-controller.adoc

```diff
@@ -337,3 +386,42 @@ The controller itself can be configured through the following environment variab
 Default: `strimzi.io/kind=cluster`.  
 * `STRIMZI_FULL_RECONCILIATION_INTERVAL` : the interval between periodic reconciliations.
 Default: 120000 ms
+ * `STRIMZI_NAMESPACE`: A comma-separated list of namespaces that the controller
+should operate in. This is a required environment variable, but the exmple YAML resources
+supplied with Strimzi use the namespace in which the controller itself is deployed as a
+default value. See the following section.
+
+[[multi-namespace]]
+==== Watching multiple namespaces
+
+The `STRIMZI_NAMESPACE` environment variable can be used to configure a single controller instance
+to operate in multiple namespaces. For each namespace given, the controller will watch for cluster ConfigMaps
+and perform periodic reconciliation. To be able to do this, the controller's ServiceAccount needs
+access to the necessary resources in those other namespaces. This can be done by creating an additional
+RoleBinding in each of those namespaces, associating the controller's ServiceAccount
+(`strimzi-cluster-controller` in the examples) with the controller's
+Role (`strimzi-controller-role` in the examples).
+
+Suppose, for example, that a controller deployed in namespace `foo` needs to operate in namespace `bar`.
+The following RoleBinding would grant the necessary permissions:
+
+.Example RoleBinding for a controller to operate in namespace `bar`
+[source,yaml,options="nowrap"]
+----
+apiVersion: v1
+kind: RoleBinding
+metadata:
+  name: strimzi-cluster-controller-binding-bar
+  namespace: bar
+  labels:
+    app: strimzi
+subjects:
+  - kind: ServiceAccount
+    name: strimzi-cluster-controller
+    namespace: foo
+roleRef:
+  kind: ClusterRole
```

Comment:
It seems that you talk about `Role` most of the time. But here you suddenly use `CluserRole`. Should this be `Role` instead? Or do we need to explain why it uses `ClusterRole` suddenly?